examples/annotation/flows/simple_annotation/flow.yaml

```
@@ -0,0 +1,41 @@
+        - Sci/Tech
+- block_type: TextParserBlock
+  block_config:
+    block_name: parse_annotation_output
+    input_cols:
+    - raw_output
+    output_cols:
+    - output
+    start_tags:
+    - ''
+    end_tags:
+    - '' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)